polylogue/cli/app.py

```
@@ -1001,7 +1027,17 @@ def _row(hit: SearchHit) -> Dict[str, Any]:
             anchor_label = f"msg-{hit.position}"
             line_hint = _find_anchor_line(Path(target_path), anchor_label)
         if target_path:
-            if open_in_editor(Path(target_path), line=line_hint):
+            target_obj = Path(target_path)
+            is_html = target_obj.suffix.lower() == ".html"
+            if is_html and anchor_label:
+                fragment = f"#${anchor_label}" if anchor_label else ""
+                try:
+                    webbrowser.open(target_obj.as_uri() + f"#{anchor_label}")
+                    ui.console.print(f"[dim]Opened {target_obj}#{anchor_label} in browser[/dim]")
+                    return
```

Ruff (0.15.9)

[warning] Consider moving this statement to an `else` block

(TRY300)